recipe/meta.yaml

```diff
@@ -50,13 +50,12 @@ source:
     folder: externals/sqlite-3.21.0.0                                                # [win]
     sha256: 95a4f2d76aeeb68b51239340e3de26e5714ecfb7c8ad82a67b17af82213a8c20         # [win]
   # Not used as of 3.7b3 (openssl-bin is), but kept for archival and license reasons.
-  - url: https://github.com/python/cpython-source-deps/archive/openssl-1.1.0h.zip    # [win]
-    folder: externals/openssl-1.1.0h                                                 # [win]
-    #sha256: f9a39b7519d3c1d5b5da7559c0b117f588eea97bc98d4ea8434d078cf5063eaa         # [win]
-    sha256: 65facaa26aad903889f1ef8e4d14542a0c0d142940a8cf47d6f57516fba6a41f         # [win]
-  - url: https://github.com/python/cpython-bin-deps/archive/openssl-bin-1.1.0h.zip   # [win]
-    folder: externals/openssl-bin-1.1.0h                                             # [win]
-    sha256: 8388d90b06b878a7fd980a46264d2d2aca2900ce36dd6834cf85861921534c35         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/openssl-1.1.0i.zip    # [win]
+    folder: externals/openssl-1.1.0i                                                 # [win]
+    sha256: 2280b9b4cba4dfcffc96d41860c79bdfd2d35945f9dd5e5a545b467ef6f96603         # [win]
+  - url: https://github.com/python/cpython-bin-deps/archive/openssl-bin-1.1.0i.zip   # [win]
+    folder: externals/openssl-bin-1.1.0i                                             # [win]
+    sha256: 71db5e71d2d6307379bac6c581492d37b65f2ad67fea207f6c0222150a4a46a7         # [win]
```

Comment:
@jjhelmus and @mariusvniekerk are we (conda-forge) good for this change? Sorry for the dummy question but I've been out of the loop in the python package for a while.

Reply:
This is a good change.  OpenSSL was updated to 1.1.0i for Windows build in the 3.7.1 release, see [bpo-34405](https://bugs.python.org/issue34405)